src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -353,15 +337,19 @@
                         (if (:catalog-facts-bash command-processing)
                           (queue/sorted-command-buffer max-enqueued)
                           max-enqueued))
+          [q message-loader] (queue/create-or-open-stockpile (conf/stockpile-dir config))
```

Comment:
Not important, and feel free to leave it alone, but I'd be tempted to call this load-messages so that it'll be `(load-messages ...)` in the invocation below.